mappings/bnfdmd/data_downloader.py

```diff
@@ -0,0 +1,129 @@
+import glob
+import os
+import re
+from datetime import datetime
+from pathlib import Path
+from urllib.parse import unquote, urljoin, urlparse
+
+import requests
+import structlog
+from bs4 import BeautifulSoup
+
+
+logger = structlog.get_logger()
+
+
+# modelled after coding_systems/base/trud_utils.py::TrudDownloader
+
+
+class Downloader:
+    def __init__(self, release_dir):
+        self.url = "https://www.nhsbsa.nhs.uk/prescription-data/understanding-our-data/bnf-snomed-mapping"
+        self.release_dir = release_dir
+
+    def get_releases(self):
+        # adapted from openprescribing/openprescribing/dmd/management/commands/fetch_bnf_snomed_mapping.py
```

Comment:
nit: As above, I'd consider providing a GitHub link to the file in question for the specific commit you borrowed from.